### PR TITLE
add vault setup initial screen and update styles based on figma design

### PR DESCRIPTION
**Pull request by zhmolly11**

### CODE REVIEW

File: frontend/src/pages/setupVault/SetupVaultView.tsx [hunk 1]
```diff
@@ -88,18 +78,63 @@ const TabbedContent: React.FC = () => {
...
-  // 6 - keygen error
+  // 0 - vault setup initial view
+  // 1 - vault setup view
+  // 2 - vault name setup
+  // 3 - keygen peer discovery screens
+  // 4 - keygen verify
+  // 5 - keygen view
+  // 6 - keygen done
+  // 7 - keygen error
   const screens = [
```
Comment (0xkobe) [inline frontend/src/pages/setupVault/SetupVaultView.tsx:89]: can you create components for each screen?

Comment (zhmolly11) [reply frontend/src/pages/setupVault/SetupVaultView.tsx:45]: fixed

**Review by 0xkobe — APPROVED** [review]
looks good to me

**Comment by johnnyluo** [comment]
resolve conflict

**Review by enriquesouza — APPROVED** [review]
LGTM

**Pull request merged**